libraries/Zigbee/src/ep/ZigbeeDimmableLight.h

```diff
@@ -0,0 +1,103 @@
+/* Class of Zigbee On/Off Light endpoint inherited from common EP class */
+
+#pragma once
+
+#include "soc/soc_caps.h"
+#if SOC_IEEE802154_SUPPORTED
+
+#include "ZigbeeEP.h"
+#include "ha/esp_zigbee_ha_standard.h"
+
+/**
+ * @brief Zigbee HA standard dimmable light device clusters.
+ *        Added here as not supported by ESP Zigbee library.
+ *
+ *
+ */
+typedef struct esp_zb_dimmable_light_cfg_s
+{
+  esp_zb_basic_cluster_cfg_t basic_cfg;       /*!<  Basic cluster configuration, @ref esp_zb_basic_cluster_cfg_s */
+  esp_zb_identify_cluster_cfg_t identify_cfg; /*!<  Identify cluster configuration, @ref esp_zb_identify_cluster_cfg_s */
+  esp_zb_groups_cluster_cfg_t groups_cfg;     /*!<  Groups cluster configuration, @ref esp_zb_groups_cluster_cfg_s */
+  esp_zb_scenes_cluster_cfg_t scenes_cfg;     /*!<  Scenes cluster configuration, @ref esp_zb_scenes_cluster_cfg_s */
+  esp_zb_on_off_cluster_cfg_t on_off_cfg;     /*!<  On off cluster configuration, @ref esp_zb_on_off_cluster_cfg_s */
+  esp_zb_level_cluster_cfg_t level_cfg;       /*!<  Level cluster configuration, @ref esp_zb_level_cluster_cfg_s */
+} esp_zb_dimmable_light_cfg_t;
+
+/**
+ * @brief Zigbee HA standard dimmable light device default config value.
+ *        Added here as not supported by ESP Zigbee library.
+ *
+ */
+#define ESP_ZB_DEFAULT_DIMMABLE_LIGHT_CONFIG()                                        \
```

Comment:
Also please rename the default config.
```suggestion
#define ZIGEEE_DEFAULT_DIMMABLE_LIGHT_CONFIG()                                        \
```

Reply:
Fixed (did use ZIGBEE_)